interview-questions.md

```diff
@@ -8,44 +8,49 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a gem?
 
-Your answer:
+Your answer: A Ruby gem is used in Ruby on Rails to hold specific information in your application 
```

Comment:
It is ok to not always know the answer (in fact, most interviewers will ask questions until you don't know something).  It is all about how you respond when you don't know.